lab_1_classify_by_unigrams/main.py

```diff
@@ -127,20 +115,28 @@ def detect_language(
     ):
         return None
 
-    profile_1_metric = compare_profiles(unknown_profile, profile_1)
-    profile_2_metric = compare_profiles(unknown_profile, profile_2)
-
-    if not (isinstance(profile_1_metric, float)
-            and isinstance(profile_2_metric, float)
-    ):
-        return None
-
-    if profile_1_metric > profile_2_metric:
-        return str(profile_2['name'])
-    if profile_1_metric < profile_2_metric:
-        return str(profile_1['name'])
-
-    return [profile_1['name'], profile_2['name']].sort()
+    first_pair = compare_profiles(
+        unknown_profile,
+        profile_1
+    )
+    second_pair = compare_profiles(
+        unknown_profile,
+        profile_2
+    )
+    name_1 = str(profile_1['name'])
+    name_2 = str(profile_2['name'])
+    if isinstance(first_pair, float) and isinstance(second_pair, float):
```

Comment:
revert the condition and fo return as soon as posible

Reply:
I can't revert it, mypy needs to be sure these are the right types, otherwise it gives errors.

Reply:
I mean you should process cornercases as soon as posible. Return None as soon as posible

Reply:
done